### DISCUSSION TITLE
Vercel%20Proxy/

### DISCUSSION THREAD

**Discussion by giscus** [ThatCoders/thatcoders.github.io#7] [Comment]
# 2022/12/18/Stack/Vercel%20Proxy/

通过反向代理--免域名备案,隐藏源主机IP,充当缓存机,免费的SSL.

https://blog.thatcoder.cn/2022/12/18/Stack/Vercel%20Proxy/

**Comment by tovarsh** [comment]
千万千万千万不要反向代理其它网站!!! 🙏🙏🙏

**Comment by weekdaycare** [comment]
我只敢反代自己的github仓库😢

**Reply by tovarsh** [reply]
我反代了github，刚代完想想不对劲，我不就是钓鱼吗。来不及关闭就收到封号邮件。

**Reply by weekdaycare** [reply]
太惨了，以后的悠着点用了_(´ཀ`」 ∠)

**Comment by easyf12** [comment]
Vercel：我这么好欺负吗？😄

**Reply by tovarsh** [reply]
那可不, 我们的V宝
